content/influxdb/v2.0/telegraf-configs/update.md

```diff
@@ -23,22 +23,23 @@ To update a Telegraf configuration, do one of the following:
 - [Customize an existing Telegraf configuration](#customize-an-existing-telegraf-configuration)
 
 ## Use the InfluxDB UI
-The InfluxDB UI lets you update the **name** or **description** of a Telegraf configuration.
-
-{{% note %}}
-You cannot modify Telegraf settings in existing Telegraf configurations directly
-through the InfluxDB UI.
-To modify existing Telegraf configuration settings,
-[use the `influx` CLI to update the Telegraf configuration](#use-the-influx-cli).
-{{% /note %}}
 
 1. In the navigation menu on the left, select **Data** (**Load Data**) > **Telegraf**.
 
     {{< nav-icon "load data" >}}
 
-2. Hover over the configuration you want to edit and click **{{< icon "pencil" >}}**
+2. To update the name or description:
+  - Hover over the configuration you want to edit and click **{{< icon "pencil" >}}**
    to update the name or description.
-3. Press **Return** or click out of the editable field to save your changes.
+  - Press **Return** or click out of the editable field to save your changes.
+4. To edit the configuration file:
+  - Click the name of the configuration.
+  - Add or update [Telegraf plugin settings](/telegraf/latest/plugins/) in the window that appears.
+  {{% note %}}
+  The text editor window doesn't detect if any plugins or settings are misconfigured. Any errors in your configuration that may cause Telegraf to fail will be displayed when you run Telegraf in the CLI.
+  {{% /note %}}
+  - Click **Save Changes** and then **Save** again to confirm.
+5. Run your updated Telegraf configuration using the `-config` flag. To find the exact command for your configuration file, click **Setup Instructions** from the **Telegraf** page.
 
```

Comment:
Need this note after (not tied to any specific step). It applies to updating your Telegraf config in the UI as a whole.

Note: Once you have updated your Telegraf configuration in the Influx UI, you must re-run the configuration. Telegraf currently does not detect changes made to a remote configuration.

Reply:
Isn't this redundant with step 5 above? It's in the steps that in order to update in the UI, the final step is to run the configuration. I'm not sure it's necessary to call it out twice.

Reply:
(ended up adding it at the top after discussing further with sam)

Reply:
this phrasing is a little strange: `Any errors in your configuration that may cause Telegraf to fail will be displayed when you run Telegraf in the CLI.`  I would propose:
`Any errors in your configuration that may cause Telegraf to fail when it is restarted with the modified configuration.`

For the most part, people run telegraf as a system service (either on Windows or Linux machines).  They do not tend to run it via a terminal window (we do that a lot for demos and when Telegraf is started it reports its status to standard out).  So, perhaps this is an opportunity to highlight where to check on the status of Telegraf via the system service logs for Linux and Windows?  I would assume that we already have this documented somewhere.  Also, it might be valuable to suggest how to "test" or "validate" the changes by attempting to manually start a Telegraf instance using this new config??  Think of hundreds of Telegraf agents which need to be restarted once the config changes....you really don't want to go through that process and then realize at the end that there was an error and have to repeat the entire process again.

Reply:
I would remove this entire section to be honest. There's really no reason to `Edit the config by downloading` anymore. If someone wanted to test their changes locally, it's probably easier to download, make the changes, then just copy and paste them back through the UI.

Maybe rename this section `Download and verify your configuration locally` or something?

Reply:
I renamed the section per Russ's suggestion -- otherwise, we don't have this download button documented anywhere.